docs/configuration/uds-operator.md

```diff
@@ -169,6 +169,17 @@ The UDS Operator uses the first `redirectUris` to populate the `match.prefix` ho
 
 For a complete example, see [app-authservice-tenant.yaml](https://github.com/defenseunicorns/uds-core/blob/main/src/test/app-authservice-tenant.yaml)
 
+#### External Session Store
+If you wish to scale Authservice horizontally, Authservice supports using an [external redis session store](https://docs.tetrate.io/istio-authservice/configuration/oidc#session-store-configuration) which can be configured by setting [UDS_AUTHSERVICE_REDIS_URI](https://github.com/defenseunicorns/uds-core/blob/main/src/pepr/zarf.yaml#L20-L22).
+
+You can also specify the `AUTHSERVICE_REDIS_URI` variable in your `uds-config.yaml`:
+
+```yaml
+variables:
+  core:
+    AUTHSERVICE_REDIS_URI: redis://redis.redis.svc.cluster.local:6379
+```
+
```

Comment:
Suspect this was added from some conflicts - I think we moved this doc to [here](https://github.com/defenseunicorns/uds-core/blob/pre-commit-lint/docs/configuration/resource-configuration-and-ha.md#authservice) so may be able to drop this section?